### DISCUSSION TITLE
aboutme/life/2016_gaokao

### DISCUSSION THREAD

**Discussion by giscus** [imarvinle/CSGuide#49] [Announcements]
# aboutme/life/2016_gaokao

编程指北: 计算机系统学习指南, 操作系统, 计算机网络, C++, Java, 算法, 数据结构

https://csguide.cn/aboutme/life/2016_gaokao.html

**Comment by caohuisheng** [comment]
高考时错的每一道选择题都是有理由的！

**Comment by 532313** [comment]
居然是川大的学长，太强了！
